workflows/mapping/config.yaml

```diff
@@ -1,9 +1,11 @@
 sampletable: pasilla_sampletable.tsv
 sample_dir: pasilla
 assembly: dm6
-data_dir: /data/LCDB/references-test
+data_dir: references-test
 
 shell_prefix: "source lcdb-workflows.bashrc; "
+cluster_default:
+  args: "--time=04:00:00"
```

Comment:
I really like this idea, and it does not look too hard to split up the cluster options if need be.